### PR TITLE
Feature class converter

### PR DESCRIPTION
**Pull request by florpadilla**
Hi team, pleaase take a look at this.

### CODE REVIEW

**Review by stospina — COMMENTED** [review]
Looks good

File: CONVERTER-SERVICE/src/converter/video.converter.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const FfmpegCommand = require('fluent-ffmpeg');
+
+//This class represents a model for convert Video into Images
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/converter/video.converter.js]: it should start in third person with a verb:
for example: // Represents a model for convert Video into Images

File: CONVERTER-SERVICE/src/converter/video.converter.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
+const FfmpegCommand = require('fluent-ffmpeg');
+
+//This class represents a model for convert Video into Images 
+class VideoConverter {
+
+  //This method allows to convert a video into an specif number of images by setting the Frames Per Second with an specific size
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/src/converter/video.converter.js]: it should start in third person with a verb:
for example: //Allows to convert a video into an specif number of images by setting the Frames Per Second with an specific size

Comment (florpadilla) [reply CONVERTER-SERVICE/src/converter/video.converter.js]: You are right! I already corrected those comments! Thanks!

File: CONVERTER-SERVICE/src/converter/video.converter.js (hunk 1 above)
Comment (florpadilla) [reply CONVERTER-SERVICE/src/converter/video.converter.js]: I already resolve that! Thanks!!

**Review by luisdavidparra — COMMENTED** [review]
All ok

**Pull request merged**